metricbeat/module/redis/_meta/supported-versions.yml

```diff
@@ -1,4 +1,9 @@
 variants:
-  - REDIS_VERSION: 3.2.12
-  - REDIS_VERSION: 4.0.11
-  - REDIS_VERSION: 5.0.5
+  - REDIS_DISTRIBUTION: oss
+    REDIS_VERSION: 3.2.12
+  - REDIS_DISTRIBUTION: oss
+    REDIS_VERSION: 4.0.11
+  - REDIS_DISTRIBUTION: oss
+    REDIS_VERSION: 5.0.5
+  - REDIS_DISTRIBUTION: enterprise
+    REDIS_VERSION: 5.4.10-22
```

Comment:
This can be tricky, this image includes redis 5.0, that seems to be the latest version of redis.

Reply:
Not relevant anymore.